src/backend/api/v1/Chats.ts

```diff
@@ -56,3 +56,10 @@ chatsRouter.post('/', function (req, res) {
         res.status(401).json({message: 'Not Authorized'})
     }
 });
+
+
+chatsRouter.get('/unreads',  async function(req, res) {
```

Comment:
Maybe add a comment about what this endpoint is returning because I needed to read through the code to understand what this returns.

Reply:
I will add it to the Google Doc. I think we need to merge them in to the code at the end.